### ISSUE TITLE
[Adaptive Card-Schema Explorer]: Narrator is giving the incorrect position for the column headers present under the table.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7906]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Screen Reader: Narrator
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Pre-Requisites:
Run the Narrator using "Control+ Windows +Enter" short key in the scan mode.

Repro Steps:
1.Open above URL with V- credential.
2.Navigate to 'Schema Explorer' link using tab key and press enter key to activate it.
3.Navigate to 'Table" link present under "Containers" from the left navigation using the tab key and activate it using Enter key.
4. Navigate to the adaptive cards present under it using the Tab key.
5. Navigate to the "showGridLines value" under "Table with gridlines" using Arrow key with narrator.
6. Observe the issue here.

Actual Result:
Narrator is announcing like "showGridLines value 3of2".

Expected Result:
Narrator shouldn't give wrong position for the options present under the table and here it should announce it as "showGridLines value 1of2".

Observation:
Issue exists throughout the table cells which was present under table adaptive cards.

User Experience:
Screen reader users will be impacted, and it might direct the user wrongly with the wrong info, if narrator announces the incorrect position for "showGridLines value".

MAS Reference:
https://liquid.microsoft.com/Web/Object/Read/ms.accessibility/Requirements/01.03.01

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/104482612/190982340-d5641040-eec7-4160-a022-4ed43b81f3fc.mp4

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by anna-dingler** [comment]
Hi @Susmitha474!

I just tested this again, and Narrator announces "Enter table: 3 by 2. ShowGridLines value".

Here, 3 by 2 is referring to the size of the table.

Is that an acceptable behavior?

**Comment by Susmitha474** [comment]
@anna-dingler I agree with your comment but once after i landed on the table narrator is providing the info regarding the table value info and after that if narrator provide some info regarding the position of the ShowGridLines value it will be helpful to the user otherwise they will get confusion as the narrator is giving the info as "3 by 2. ShowGridLines value" &next "Behavior 2of 2".

**Comment by anna-dingler** [comment]
Hey @Susmitha474, thank you for the quick reply!

I've been investigating this further, and I believe this is a Narrator bug. 

I confirmed that our team has added the necessary aria labels as outlined here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role

I then confirmed that the same behavior (not announcing the column and row location) occurs on the examples from the site above and on the `<table/>` element examples (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table). However, NVDA is announcing the table element properly, and both Narrator and NVDA work well with Chrome browser.

I'm closing out this bug and will follow up with the Narrator team. Please let me know if you have any questions!
